external-resolver/src/main/scala/org/genivi/sota/resolver/vehicles/VehicleRepository.scala

```diff
@@ -52,16 +57,72 @@ object VehicleRepository {
 
   val installedPackages = TableQuery[InstalledPackageTable]
 
-  def installPackage(vin: Vehicle.Vin, pkgId: Package.Id): DBIO[Int] =
-    installedPackages.insertOrUpdate((vin, pkgId))
+  def installPackage
+    (vin: Vehicle.Vin, pkgId: Package.Id)
+    (implicit db: Database, ec: ExecutionContext): DBIO[Unit] =
```

Comment:
My initial idea of keeping Repository simple and doing all the work inside Future in Functions doesn't seem scale. Interesting stuff will need to happen in DBIO, so then it might as well be grouped with the other DBIO stuff in Repository? Maybe we can discuss this once you are done with the meeting?